### PR TITLE
Update to cucumber 4.5

### PR DESCRIPTION
**Pull request by beardedqa**
As after cucumber some imports have been changed, change in documentation is required. 

**Please do not merge this change** before the IntelliJ cucumber java plugin update https://youtrack.jetbrains.com/issue/IDEA-217391

### CODE REVIEW

**Comment by luke-hill** [comment]
@beardedqa - Are you happy for this to be merged in now?

I'd rather a java person give it the once over as well, but you mentioned about waiting until the plugin update. The thread seems to elude to the fact this is done.

File: data/versions.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@
 #
 
 # Note: When updating cucumber-jvm version, please update both the snapshot version and the regular version
-cucumberjvm-snapshot: "4.4.0-SNAPSHOT"
-cucumberjvm: "4.3.1"
+cucumberjvm-snapshot: "4.5.5-SNAPSHOT"
```
Comment (mlvandijk) [inline data/versions.yaml]: I'd like to at least merge the version updates (if not the import statements just yet). Would you mind separating these changes into 2 different PRs? (We could update import statements when v5 hits? And we'd need to make sure to update consistently all throughout the docs)

**Review by mlvandijk — CHANGES_REQUESTED** [review]

File: data/versions.yaml (hunk 1 above)
Comment (mpkorstanje) [reply data/versions.yaml]: Better make a new PR then. We're at 4.7.1 right now.

Comment (mpkorstanje) [reply data/versions.yaml]: With 5.0.0-SNAPSHOT as the next planned release.

Comment (mlvandijk) [reply data/versions.yaml]: Good to know, thnx :)

**Comment by luke-hill** [comment]
@beardedqa can you drop the 2nd commit and then rebase the 1 remaining commit.

**Comment by mlvandijk** [comment]
@beardedqa Would you mind removing the version change from this PR? We could then merge the other changes. (Would love help checking the rest of the docs for changes needed btw)

**Comment by mpkorstanje** [comment]
I've merged it for now. With v5.0.0-RC1 coming up I'd rather not wait any longer for IDEA to catch up.

Cheers!

**Pull request merged**